src/rail/pipelines/utility/apply_phot_errors.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Prerquisites, os, and numpy
+import os
+import numpy as np
+
+# Various rail modules
+from rail.tools.photometry_tools import Dereddener, Reddener
+
+from rail.core.stage import RailStage, RailPipeline
+
+import ceci
+
+from rail.core.utils import RAILDIR
+
+if 'PZ_DUSTMAP_DIR' not in os.environ:
+    os.environ['PZ_DUSTMAP_DIR'] = '.'
+
+dustmap_dir = os.path.expandvars("${PZ_DUSTMAP_DIR}")
+
+
+ERROR_MODELS = dict(
+    lsst = dict(
+        ErrorModel='LSSTErrorModel',
+        Module='rail.creation.degraders.photometric_errors',
+    ),
+    roman = dict(
+        ErrorModel='RomanErrorModel',
+        Module='rail.creation.degraders.photometric_errors',
+    ),
+    #euclid = dict(
+    #    ErrorModel='EuclidErrorModel',
+    #    Module='rail.creation.degraders.photometric_errors',
+    #),
```

Comment:
Is Euclid supposed to be commented out?

Reply:
For now, yes..

Reply:
(We don't have data with roman and rubin and euclid all in the same file)